.prow/features.yaml

```diff
@@ -180,6 +182,49 @@ presubmits:
           env:
             - name: CNI
               value: canal
+            - name: PROVIDER
+              value: all
+            - name: VERSION_TO_TEST
+              value: v1.23.8
+            - name: KUBERMATIC_EDITION
+              value: ee
+            - name: SERVICE_ACCOUNT_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: e2e-ci
+                  key: serviceAccountSigningKey
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              memory: 4Gi
+              cpu: 2
+            limits:
+              memory: 6Gi
+
+  - name: pre-kubermatic-dualstack-e2e-smoke
+    run_if_changed: ".*(cilium|canal|dualstack|api|addon|defaults|cni|validation|operator|provider|machine|webhook|crd|.prow|go.mod|proxy|network).*"
```

Comment:
Do we really want something this broad? Because in general, I think this will almost always run.

Reply:
We probably don't need all of that. Since the feature is new I have included anything that touches networking. Definitely can be improved in the future PRs.

Reply:
I think this PR would be the best candidate to perform that improvement. Not sure why this should be done in some "future" PR.